app/models/cafe_worker.rb

```diff
@@ -0,0 +1,40 @@
+class CafeWorker < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :cafe_shift, inverse_of: :cafe_worker
+
+  validates :user_id, :cafe_shift_id, presence: true
+  validate :user_attributes?
+
+  scope :shift_lp_year, -> (lp, year) do
+    joins(:cafe_shift).where(cafe_shifts: { lp: lp })
+    .where('cafe_shifts.start > ?', year.beginning_of_year)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.